<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthCare</class>
 <widget class="QWidget" name="HealthCare">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>997</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>32</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777150</width>
    <height>16777211</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>HealthCare</string>
  </property>
  <property name="windowOpacity">
   <double>44.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-6</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:'#3c4043';
border-radius:25px;</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>890</y>
     <width>251</width>
     <height>84</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid ;
border-radius: 9px;
padding:10px;
background-color:&quot;#3e91ff&quot;;
color:&quot;white&quot;;</string>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
   <property name="autoRepeatDelay">
    <number>294</number>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="exercise_name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>3</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;white&quot;;</string>
   </property>
   <property name="text">
    <string>운동이름</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>0</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;white&quot;;</string>
   </property>
   <property name="text">
    <string>사용자</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="exercise_cam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>801</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="user_cam">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>800</width>
     <height>901</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-16</number>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid; color:'white';</string>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>70</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777210</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;
border-radius: 9px;
background-color:&quot;#DADCE0&quot;;
padding:10px;</string>
   </property>
   <property name="lineWidth">
    <number>13</number>
   </property>
   <property name="text">
    <string>정확도</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>300</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;
border-radius: 9px;
background-color:&quot;#DADCE0&quot;;
padding:10px;</string>
   </property>
   <property name="text">
    <string>난이도</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="exercise_score">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>150</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>80</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:&quot;#DADCE0&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="exercise_level">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>360</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:&quot;#DADCE0&quot;;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>971</y>
     <width>800</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:'white';color:'black';</string>
   </property>
   <property name="text">
    <string>운동을 시작해주세요.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
